### DISCUSSION TITLE
Welcome to Starter Discussions!

### DISCUSSION THREAD

**Discussion by vinzvinci** [WebXDAO/start-here#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Superblue045** [comment]
Hi everyone. I am Hiroshi. Nice to meet u. I'm new here as well.

**Reply by Goutamxd** [reply]
same here my friend:)
